src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: How to watch Edge Orchestrator logs
+description: Check the edge node and the edge services triggers logs.
+meta_tags: deploy, orchestrator, logs
+namespace: docs_deploy_watch_logs
+menu_namespace: deployMenu
+permalink: /documentation/products/guides/deploy/watch-logs/
+---
+
+import Button from '~/components/Button.astro'
+
+After the installation of Edge Orchestrator Agent, you can check the [edge node's logs](#checking-an-edge-nodes-logs) and the [edge services triggers' logs](#checking-edge-services-logs).
+
+--- 
+
+## Checking an edge node's logs
+
+1. Access your edge node.
+2. Open the terminal. 
+3. Run the following command: 
+
+```bash
+sudo vim /var/log/azion/edge-orchestrator.log 
+```
+
+:::note 
+You can choose the text editor you want, as long as it's installed on the edge node.
```

Comment:
How about adding an example command to install vim on an edge node? Or link to a guide on installing packages in edge nodes?